test/utils/update/mocks.ts

```diff
@@ -304,7 +304,7 @@ export const FETCH_INDIVIDUAL_TRUSTEE_DATA_MOCK: IndividualTrusteeData[] = [
     dateOfBirth: "1988-12-01",
     nationality: "German",
     corporateIndicator: "N",
-    trusteeTypeId: "5001",
+    trusteeTypeId: "5002",
```

Comment:
Wondering why the 5001 value had to change to 5002 here? There seems to already be some example data with the 5002 code (see line 265).

Also, now no example with the 5001 code in this test file - is that (or could that be) a problem?

Reply:
I changed it to 5002 as the expected value is Beneficiary. The code was incorrectly mapping both 5001 and 5002 to Beneficiary before the code fix. The alternative is to keep it at 5001 and change the expected value to Beneficial Owner - although we are testing this mapping in other tests.

Reply:
The code was incorrectly mapping 5001 to type Beneficiary. 5002 is the correct code for Beneficiary so no need to revert.

Reply:
Resolving this as 5001 no longer a valid code and test for such an invalid code is covered below.